pycbc/events/ranking.py

```diff
@@ -4,6 +4,49 @@
 import numpy
 
 
+def newsnr_sgveto_psdvar_scaled(snr, bchisq, sgchisq, psd_var_val):
+    """ Combined SNR derived from NewSNR, Sine-Gaussian Chisq and PSD
+    variation statistic """
+    nsnr = numpy.array(newsnr_sgveto(snr, bchisq, sgchisq), ndmin=1)
+    psd_var_val = numpy.array(psd_var_val, ndmin=1)
+    # 1.2 is the expected maximum psd_var_val over gaussian noise.
```

Comment:
Would be nice to expose the hard-code parameters, even if only as kwargs here.

Reply:
Seems that the 1.2 number isn't actually used in the stat. So remove the comment